Automattic/crowdsignal-ui
# Add Embed block support

## Summary

The purpose of this PR is to add support for the WP Embed Block.

Depends on: D74885-code
Ref: c/ellkRtk8-tr

## Test Instructions

* Open o create a project
* Paste a URL from one of the supported providers or manually add the Embed Block and update the URL field
* The embed content should be loaded correctly
